mlflow/lightgbm.py

```diff
@@ -424,7 +429,32 @@ def log_feature_importance_plot(features, importance, importance_type):
             finally:
                 shutil.rmtree(tmpdir)
 
-        try_mlflow_log(log_model, model, artifact_path="model")
+        input_example = None
+        signature = None
+        try:
+            raw_data = train_data.get_data()
+            if isinstance(raw_data, str):
+                raise Exception(
+                    "The training data was given as a path. Input example and"
+                    + " model signature inference is not supported for datasets specified by paths."
+                )
+
+            SAMPLE_ROWS = 5
```

Comment:
@dbczumar should we put this constant in a separate file (for example `autologging_utils.py`) and reuse it across autlogging integrations?